src/main/java/com/denys/toukdemo/dto/Reservation.java

```diff
@@ -0,0 +1,38 @@
+package com.denys.toukdemo.dto;
+
+import javax.validation.Valid;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Reservation {
+
+    @Valid
```

Comment:
kto odpala tę walidacje?

Reply:
> kto odpala tę walidacje?

to była podjęta próba automatycznej walidacji przy wywołaniu endpointa w kontrollerze
nie udało się zrobić to szybko więc zrobiłem klasą UserValidator a to przez nieuważność zapomniałęm usunąć